ApollosProject/apollos-apps
# fix: revert to Component

### **Purpose**
The purpose of this PR is to ensure the items in the HorizontalTileFeed display correctly. 

### **Solution**
Revert ContentCardConnected to use `Component` instead of `DefaultCard`.

### **Testing**
**Setup**
1. Clone the repo and checkout the following branch: `revert-to-component`
2. Pick any multi-tenant app (I'm using crossings_community_church) for a fresh spin up and run the setup script (`setup-app.sh`)
3. Run the iOS or Android app

**Steps to Test**
1. Login
2. Select any video item that features a HorizontalTileFeed
3. The items in the TileFeed should display normally

### **Before**


https://user-images.githubusercontent.com/19194337/184691291-80a6f588-668f-42c4-a737-a3b4ba58e62b.mov

### **After**

https://user-images.githubusercontent.com/19194337/184691348-918093d9-bf75-4599-8eff-189a785db9ea.mov
